libcaf_core/caf/mixin/requester.hpp

```diff
@@ -23,7 +23,9 @@
 namespace caf::mixin {
 
 /// A `requester` is an actor that supports
+/// @code
 /// `self->request(...).{then|await|receive}`.
+/// @endcode
```

Comment:
Why add the `code` block here?

Reply:
Doxygen expects a `code` block here for the code as it is in a separate line. Removing it shows a warning.